src/components/Entities/EntityExecutions.tsx

```diff
@@ -48,21 +52,29 @@ export const EntityExecutions: React.FC<EntityExecutionsProps> = ({
         [id, resourceType]
     );
 
+    const allFilters = compact([
+        ...baseFilters,
+        ...filtersState.appliedFilters,
+        archivedFilter.getFilter()
+    ]);
     const executions = useWorkflowExecutions(
         { domain, project },
         {
             sort,
-            filter: [...baseFilters, ...filtersState.appliedFilters],
+            filter: allFilters,
             limit: 100
         }
     );
 
-    if (chartIds.length > 0)
+    if (chartIds.length > 0) {
         executions.value = executions.value.filter(item =>
             chartIds.includes(item.id.name)
         );
+    }
+
     /** Don't render component until finish fetching user profile */
-    if (filtersState.filters[4].status !== 'LOADED') {
```

Comment:
Should we make this a constant rather than inline string?

Reply:
Yep, I can use  `fetchStates.LOADED` instead.

Reply:
done